packages/nmr-simulation/src/SpinSystem.js

```diff
@@ -32,13 +32,14 @@ export default class SpinSystem {
       for (j = 0; j < nCoup; j++) {
         var withID = tokens[4 + 3 * j] - 1;
         var idx = ids[withID];
-        jc[i][idx] = +tokens[6 + 3 * j];
+        // jc[i][idx] = +tokens[6 + 3 * j];
+        jc.set(i, idx, jc.get(i, idx)+tokens[6 + 3 * j]);
```

Comment:
I don't know which one is correct but this is not the same as before

Reply:
We would rather use `Number(tokens[6 + 3 * j]);` rather than `+tokens[6 + 3 * j];` in the future to avoid any confusion.

Reply:
To be correct, in the symmetrization 5 lines bellow, you should put the average of the i, j and j, i values, if both are different than 0, else, it must use the Max between both values.